homeassistant/components/todoist/calendar.py

```diff
@@ -381,6 +389,16 @@ def __init__(
         else:
             self._project_id_whitelist = []
 
+    def _parse_due_date(self, data: dict) -> datetime:
```

Comment:
Dates can be in 3 formats:
`'2019-10-01'`
`'2019-10-01T12:00:00'`
`'2019-10-01T12:00:00Z'`